Content.Server/Administration/Rejuvenate.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Content.Server.GameObjects;
+using Content.Server.Players;
+using Robust.Server.Interfaces.Console;
+using Robust.Server.Interfaces.Player;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.IoC;
+
+namespace Content.Server.Administration
+{
+    class Rejuvenate : IClientCommand
+    {
+        public string Command => "rejuvenate";
+        public string Description => "Fully heals a mob.";
+        public string Help => "rejuvenate <mobUid_1> <mobUid_2> ... <mobUid_n>";
+
+        public void Execute(IConsoleShell shell, IPlayerSession player, string[] args)
+        {
+            if (player == null)
```

Comment:
`player` session is always null if this command was ran from the server's local console (the cmd window). Since this specific command does not actually use the `player` as the target of any action, there is no need to prevent the server console from executing this command.